src/controllers/room.controller.js

```diff
@@ -0,0 +1,40 @@
+const roomService = require('../services/room.service');
+const messageService = require('../services/message.service');
+
+async function create(req, res) {
+  const { name } = req.body;
+  const room = await roomService.create(name);
+
+  res.status(201).send(room);
+}
+
+async function join(req, res) {
+  const { id } = req.params;
+  const room = await roomService.join(id);
+  const messages = await messageService.getMessages(id);
+
+  res.send(room, messages);
```

Comment:
The `res.send` method should only take one argument. If you want to send both `room` and `messages`, consider combining them into an object, like `res.send({ room, messages })`.